test/e2e/utils_test.go

```diff
@@ -155,22 +159,26 @@ func checkIfRemovedWorkResourcesFromAllMemberClusters() {
 // cleanupCRP deletes the CRP and waits until the resources are not found.
 func cleanupCRP(name string) {
 	// TODO(Arvindthiru): There is a conflict which requires the Eventually block, not sure of series of operations that leads to it yet.
-	Eventually(func(g Gomega) error {
+	Eventually(func() error {
```

Comment:
Note that this is actually the supported pattern in Ginkgo.

Reply:
oh, the fix is to remove the g.Expect(err) in the Eventually, which looks not correct.

Reply:
```
Eventually can also accept functions that take a Gomega parameter. These functions are then passed a local Gomega that can be used to make assertions inside the function as it is polled. Eventually will retry the function if an assertion fails. This would allow us to replace:
```

https://onsi.github.io/ginkgo/

Reply:
i see, good to know this pattern.

I think we need to either use the "local gomega + return void" or "no local gomega + return error".

The current code is mixed.
The example is using local gomega to replace the "return error".